web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -49,6 +50,12 @@ export namespace TaskItem {
               <md-icon name="incoming-call-active_16"></md-icon>
             </md-badge>
           `;
+          case "midcall telephony":
```

Comment:
TaskItem is a generic component. it is not aware of contact center specific thing. like midcall, telephony. should not add desktop specific code here